idaes/apps/uncertainty_propagation/tests/test_uncertainties.py

```diff
@@ -12,26 +12,24 @@
 #################################################################################
 import sys
 import os
-from unittest.mock import patch
-
-sys.path.append(os.path.abspath(".."))  # current folder is ~/tests
 import numpy as np
 import pandas as pd
-from scipy import sparse
 import pytest
-from pytest import approx
+
+from pyomo.environ import *
+import pyomo.contrib.parmest.parmest as parmest
+
 from idaes.apps.uncertainty_propagation.uncertainties import (
     quantify_propagate_uncertainty,
     propagate_uncertainty,
     clean_variable_name,
 )
-from pyomo.opt import SolverFactory
-from pyomo.environ import *
-import pyomo.contrib.parmest.parmest as parmest
 
-ipopt_available = SolverFactory("ipopt").available()
-kaug_available = SolverFactory("k_aug").available()
-dotsens_available = SolverFactory("dot_sens").available()
+sys.path.append(os.path.abspath(".."))  # current folder is ~/tests
```

Comment:
Why are we doing this?  I suspect it may be left from someone doing development without installing idaes?

Reply:
Looking at the whole file now, I think this is probably not needed.